Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -111,15 +111,14 @@ def get_incident_labels_map(labels):
 def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, incident_id, hours_back, ignore_closed,
                           max_number_of_results, extra_query, applied_condition):
     condition_string = ' %s ' % applied_condition.lower()
-    similar_keys_query = condition_string.join(
-        map(lambda t: '%s:"%s"' % (t[0], t[1]
-                                   .replace('\\', '\\\\')
-                                   .replace('"', r'\"')
-                                   .replace("\n", "\\n")
-                                   .replace("\r", "\\r")
-                                   if isinstance(t[1], STRING_TYPES) else t[1]
-                                   ),
-            similar_incident_keys.items()))
+
+    similar_keys_query = []
+    for key in similar_incident_keys.items():
+        compare_str = '%s:=%s' if str(key[1]).isdigit() else '%s="%s"'
+        similar_keys_query.append(compare_str % (key[0], str(key[1]).replace('"', r'\"')
+                                                 .replace("\n", "\\n").replace("\r", "\\r")))
```

Comment:
don't use `%` string formatting, switch to `.format`.
keep all `replace` calls as in the original code (in separated lines)